index.bs

```diff
@@ -2267,6 +2273,8 @@ during registration.
     interface AuthenticatorAttestationResponse : AuthenticatorResponse {
         [SameObject] readonly attribute ArrayBuffer      attestationObject;
         sequence<DOMString>                              getTransports();
+        ArrayBuffer                                      getAuthenticatorData();
+        DOMString                                        getPublicKey();
```

Comment:
I'm in favor, with the commentors below, that this be a nullable ArrayBuffer with the DER-encoded bytes of the SPKI.

```suggestion
        ArrayBuffer?                                        getPublicKey();
```